src/rules/no-standalone-expect.ts

```diff
@@ -0,0 +1,66 @@
+import { createRule, isDescribe, isExpectCall, isFunction } from './tsUtils';
+import { TSESTree } from '@typescript-eslint/experimental-utils';
+
+const getBlockType = (stmt: TSESTree.BlockStatement) => {
+  const func = stmt.parent;
+  // functionDeclaration: function func() {}
+  if (func && func.type === 'FunctionDeclaration') {
+    return 'function';
+  } else if (func && isFunction(func) && func.parent) {
+    const expr = func.parent;
+    // arrowfunction or function expr
+    if (expr.type === 'VariableDeclarator') {
+      return 'function';
+      // if it's not a variable, it will be callExpr, we only care about describe
+    } else if (isDescribe(expr as TSESTree.CallExpression)) {
```

Comment:
Is it possible to eliminate this cast?

Reply:
`(expr.type === 'CallExpression && isDescribe(expr))` should do the trick

Reply:
`if (expr.type === AST_NODE_TYPES .CallExpression && isDescribe(expr))` 😉 

(hmm someone should make an eslint plugin for that... 🤔)

Reply:
yeah that works. Thanks, not super familiar with typescript